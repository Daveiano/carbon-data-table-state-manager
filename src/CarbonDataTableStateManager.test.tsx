import React from "react";
import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";

import CarbonDataTableStateManager from "./index";
import {
  rowsMany,
  columns,
  sortInfo,
  rows,
  rowsSelected,
} from "../stories/data";

describe("Running Test for CarbonDataTableStateManager Button", () => {
  test("test", () => {
    const { container } = render(
      <CarbonDataTableStateManager
        pageSizes={[10, 25, 50, 100, 200, 500]}
        sortInfo={sortInfo}
        columns={columns}
        rows={rows}
        pageSize={5}
        start={0}
      />
    );

    expect(container).toMatchSnapshot();
  });
});
