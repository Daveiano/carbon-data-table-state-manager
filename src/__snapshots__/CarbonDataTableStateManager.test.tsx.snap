// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Running Test for CarbonDataTableStateManager Button test 1`] = `
<div>
  <div
    class="bx--data-table-container"
  >
    <div
      class="bx--data-table-content"
    >
      <table
        class="bx--data-table bx--data-table--normal bx--data-table--sort bx--data-table--no-border"
      >
        <thead>
          <tr>
            <th
              aria-sort="none"
              scope="col"
            >
              <div
                id="table-sort-1"
                style="display: none;"
              >
                Click to sort rows by [object Object] header in ascending order
              </div>
              <button
                aria-describedby="table-sort-1"
                class="bx--table-sort"
                data-column-id="time"
                data-sort-cycle="tri-states-from-ascending"
                data-sort-direction="NONE"
                type="button"
              >
                <span
                  class="bx--table-sort__flex"
                >
                  <div
                    class="bx--table-header-label"
                  >
                    <div
                      style="display: flex; align-items: center; justify-content: space-between;"
                    >
                      <div
                        style="margin-right: 20px;"
                      >
                        Time
                         
                        <br />
                        <span
                          class="bx--type-helper-text-01"
                        />
                      </div>
                      <button
                        aria-describedby="icon-tooltip-1"
                        class="bx--tooltip__trigger bx--tooltip--a11y bx--tooltip--bottom bx--tooltip--align-start"
                        style="cursor: default;"
                        type="button"
                      >
                        <span
                          class="bx--assistive-text"
                          id="icon-tooltip-1"
                        >
                          Date format is YYYY/MM/DD HH:mm
                        </span>
                        <svg
                          aria-hidden="true"
                          fill="currentColor"
                          focusable="false"
                          height="16"
                          preserveAspectRatio="xMidYMid meet"
                          viewBox="0 0 16 16"
                          width="16"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M8.5 11L8.5 6.5 6.5 6.5 6.5 7.5 7.5 7.5 7.5 11 6 11 6 12 10 12 10 11zM8 3.5c-.4 0-.8.3-.8.8S7.6 5 8 5c.4 0 .8-.3.8-.8S8.4 3.5 8 3.5z"
                          />
                          <path
                            d="M8,15c-3.9,0-7-3.1-7-7s3.1-7,7-7s7,3.1,7,7S11.9,15,8,15z M8,2C4.7,2,2,4.7,2,8s2.7,6,6,6s6-2.7,6-6S11.3,2,8,2z"
                          />
                        </svg>
                      </button>
                    </div>
                  </div>
                  <svg
                    aria-hidden="true"
                    class="bx--table-sort__icon"
                    fill="currentColor"
                    focusable="false"
                    height="20"
                    preserveAspectRatio="xMidYMid meet"
                    viewBox="0 0 32 32"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M16 4L6 14 7.41 15.41 15 7.83 15 28 17 28 17 7.83 24.59 15.41 26 14 16 4z"
                    />
                  </svg>
                  <svg
                    aria-hidden="true"
                    class="bx--table-sort__icon-unsorted"
                    fill="currentColor"
                    focusable="false"
                    height="20"
                    preserveAspectRatio="xMidYMid meet"
                    viewBox="0 0 32 32"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M27.6 20.6L24 24.2 24 4 22 4 22 24.2 18.4 20.6 17 22 23 28 29 22zM9 4L3 10 4.4 11.4 8 7.8 8 28 10 28 10 7.8 13.6 11.4 15 10z"
                    />
                  </svg>
                </span>
              </button>
            </th>
            <th
              aria-sort="none"
              scope="col"
            >
              <div
                id="table-sort-2"
                style="display: none;"
              >
                Click to sort rows by [object Object] header in ascending order
              </div>
              <button
                aria-describedby="table-sort-2"
                class="bx--table-sort"
                data-column-id="temperature"
                data-sort-cycle="tri-states-from-ascending"
                data-sort-direction="NONE"
                type="button"
              >
                <span
                  class="bx--table-sort__flex"
                >
                  <div
                    class="bx--table-header-label"
                  >
                    <div
                      style="display: flex; align-items: center; justify-content: space-between;"
                    >
                      <div
                        style="margin-right: 20px;"
                      >
                        Temperature
                         
                        <br />
                        <span
                          class="bx--type-helper-text-01"
                        >
                          in °C
                        </span>
                      </div>
                    </div>
                  </div>
                  <svg
                    aria-hidden="true"
                    class="bx--table-sort__icon"
                    fill="currentColor"
                    focusable="false"
                    height="20"
                    preserveAspectRatio="xMidYMid meet"
                    viewBox="0 0 32 32"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M16 4L6 14 7.41 15.41 15 7.83 15 28 17 28 17 7.83 24.59 15.41 26 14 16 4z"
                    />
                  </svg>
                  <svg
                    aria-hidden="true"
                    class="bx--table-sort__icon-unsorted"
                    fill="currentColor"
                    focusable="false"
                    height="20"
                    preserveAspectRatio="xMidYMid meet"
                    viewBox="0 0 32 32"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M27.6 20.6L24 24.2 24 4 22 4 22 24.2 18.4 20.6 17 22 23 28 29 22zM9 4L3 10 4.4 11.4 8 7.8 8 28 10 28 10 7.8 13.6 11.4 15 10z"
                    />
                  </svg>
                </span>
              </button>
            </th>
            <th
              aria-sort="none"
              scope="col"
            >
              <div
                id="table-sort-3"
                style="display: none;"
              >
                Click to sort rows by [object Object] header in ascending order
              </div>
              <button
                aria-describedby="table-sort-3"
                class="bx--table-sort"
                data-column-id="felt_temperature"
                data-sort-cycle="tri-states-from-ascending"
                data-sort-direction="NONE"
                type="button"
              >
                <span
                  class="bx--table-sort__flex"
                >
                  <div
                    class="bx--table-header-label"
                  >
                    <div
                      style="display: flex; align-items: center; justify-content: space-between;"
                    >
                      <div
                        style="margin-right: 20px;"
                      >
                        Felt temperature
                         
                        <br />
                        <span
                          class="bx--type-helper-text-01"
                        >
                          in °C
                        </span>
                      </div>
                      <button
                        aria-describedby="icon-tooltip-2"
                        class="bx--tooltip__trigger bx--tooltip--a11y bx--tooltip--bottom bx--tooltip--align-start"
                        style="cursor: default;"
                        type="button"
                      >
                        <span
                          class="bx--assistive-text"
                          id="icon-tooltip-2"
                        >
                          Some tooltip.
                        </span>
                        <svg
                          aria-hidden="true"
                          fill="currentColor"
                          focusable="false"
                          height="16"
                          preserveAspectRatio="xMidYMid meet"
                          viewBox="0 0 16 16"
                          width="16"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M8.5 11L8.5 6.5 6.5 6.5 6.5 7.5 7.5 7.5 7.5 11 6 11 6 12 10 12 10 11zM8 3.5c-.4 0-.8.3-.8.8S7.6 5 8 5c.4 0 .8-.3.8-.8S8.4 3.5 8 3.5z"
                          />
                          <path
                            d="M8,15c-3.9,0-7-3.1-7-7s3.1-7,7-7s7,3.1,7,7S11.9,15,8,15z M8,2C4.7,2,2,4.7,2,8s2.7,6,6,6s6-2.7,6-6S11.3,2,8,2z"
                          />
                        </svg>
                      </button>
                    </div>
                  </div>
                  <svg
                    aria-hidden="true"
                    class="bx--table-sort__icon"
                    fill="currentColor"
                    focusable="false"
                    height="20"
                    preserveAspectRatio="xMidYMid meet"
                    viewBox="0 0 32 32"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M16 4L6 14 7.41 15.41 15 7.83 15 28 17 28 17 7.83 24.59 15.41 26 14 16 4z"
                    />
                  </svg>
                  <svg
                    aria-hidden="true"
                    class="bx--table-sort__icon-unsorted"
                    fill="currentColor"
                    focusable="false"
                    height="20"
                    preserveAspectRatio="xMidYMid meet"
                    viewBox="0 0 32 32"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M27.6 20.6L24 24.2 24 4 22 4 22 24.2 18.4 20.6 17 22 23 28 29 22zM9 4L3 10 4.4 11.4 8 7.8 8 28 10 28 10 7.8 13.6 11.4 15 10z"
                    />
                  </svg>
                </span>
              </button>
            </th>
            <th
              aria-sort="none"
              scope="col"
            >
              <div
                id="table-sort-4"
                style="display: none;"
              >
                Click to sort rows by [object Object] header in ascending order
              </div>
              <button
                aria-describedby="table-sort-4"
                class="bx--table-sort"
                data-column-id="dew_point"
                data-sort-cycle="tri-states-from-ascending"
                data-sort-direction="NONE"
                type="button"
              >
                <span
                  class="bx--table-sort__flex"
                >
                  <div
                    class="bx--table-header-label"
                  >
                    <div
                      style="display: flex; align-items: center; justify-content: space-between;"
                    >
                      <div
                        style="margin-right: 20px;"
                      >
                        Dew point
                         
                        <br />
                        <span
                          class="bx--type-helper-text-01"
                        >
                          in °C
                        </span>
                      </div>
                    </div>
                  </div>
                  <svg
                    aria-hidden="true"
                    class="bx--table-sort__icon"
                    fill="currentColor"
                    focusable="false"
                    height="20"
                    preserveAspectRatio="xMidYMid meet"
                    viewBox="0 0 32 32"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M16 4L6 14 7.41 15.41 15 7.83 15 28 17 28 17 7.83 24.59 15.41 26 14 16 4z"
                    />
                  </svg>
                  <svg
                    aria-hidden="true"
                    class="bx--table-sort__icon-unsorted"
                    fill="currentColor"
                    focusable="false"
                    height="20"
                    preserveAspectRatio="xMidYMid meet"
                    viewBox="0 0 32 32"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M27.6 20.6L24 24.2 24 4 22 4 22 24.2 18.4 20.6 17 22 23 28 29 22zM9 4L3 10 4.4 11.4 8 7.8 8 28 10 28 10 7.8 13.6 11.4 15 10z"
                    />
                  </svg>
                </span>
              </button>
            </th>
            <th
              aria-sort="none"
              scope="col"
            >
              <div
                id="table-sort-5"
                style="display: none;"
              >
                Click to sort rows by [object Object] header in ascending order
              </div>
              <button
                aria-describedby="table-sort-5"
                class="bx--table-sort"
                data-column-id="pressure"
                data-sort-cycle="tri-states-from-ascending"
                data-sort-direction="NONE"
                type="button"
              >
                <span
                  class="bx--table-sort__flex"
                >
                  <div
                    class="bx--table-header-label"
                  >
                    <div
                      style="display: flex; align-items: center; justify-content: space-between;"
                    >
                      <div
                        style="margin-right: 20px;"
                      >
                        Pressure
                         
                        <br />
                        <span
                          class="bx--type-helper-text-01"
                        >
                          in hPa
                        </span>
                      </div>
                    </div>
                  </div>
                  <svg
                    aria-hidden="true"
                    class="bx--table-sort__icon"
                    fill="currentColor"
                    focusable="false"
                    height="20"
                    preserveAspectRatio="xMidYMid meet"
                    viewBox="0 0 32 32"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M16 4L6 14 7.41 15.41 15 7.83 15 28 17 28 17 7.83 24.59 15.41 26 14 16 4z"
                    />
                  </svg>
                  <svg
                    aria-hidden="true"
                    class="bx--table-sort__icon-unsorted"
                    fill="currentColor"
                    focusable="false"
                    height="20"
                    preserveAspectRatio="xMidYMid meet"
                    viewBox="0 0 32 32"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M27.6 20.6L24 24.2 24 4 22 4 22 24.2 18.4 20.6 17 22 23 28 29 22zM9 4L3 10 4.4 11.4 8 7.8 8 28 10 28 10 7.8 13.6 11.4 15 10z"
                    />
                  </svg>
                </span>
              </button>
            </th>
            <th
              aria-sort="none"
              scope="col"
            >
              <div
                id="table-sort-6"
                style="display: none;"
              >
                Click to sort rows by [object Object] header in ascending order
              </div>
              <button
                aria-describedby="table-sort-6"
                class="bx--table-sort"
                data-column-id="humidity"
                data-sort-cycle="tri-states-from-ascending"
                data-sort-direction="NONE"
                type="button"
              >
                <span
                  class="bx--table-sort__flex"
                >
                  <div
                    class="bx--table-header-label"
                  >
                    <div
                      style="display: flex; align-items: center; justify-content: space-between;"
                    >
                      <div
                        style="margin-right: 20px;"
                      >
                        Humidity
                         
                        <br />
                        <span
                          class="bx--type-helper-text-01"
                        >
                          in %
                        </span>
                      </div>
                    </div>
                  </div>
                  <svg
                    aria-hidden="true"
                    class="bx--table-sort__icon"
                    fill="currentColor"
                    focusable="false"
                    height="20"
                    preserveAspectRatio="xMidYMid meet"
                    viewBox="0 0 32 32"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M16 4L6 14 7.41 15.41 15 7.83 15 28 17 28 17 7.83 24.59 15.41 26 14 16 4z"
                    />
                  </svg>
                  <svg
                    aria-hidden="true"
                    class="bx--table-sort__icon-unsorted"
                    fill="currentColor"
                    focusable="false"
                    height="20"
                    preserveAspectRatio="xMidYMid meet"
                    viewBox="0 0 32 32"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M27.6 20.6L24 24.2 24 4 22 4 22 24.2 18.4 20.6 17 22 23 28 29 22zM9 4L3 10 4.4 11.4 8 7.8 8 28 10 28 10 7.8 13.6 11.4 15 10z"
                    />
                  </svg>
                </span>
              </button>
            </th>
            <th
              aria-sort="none"
              scope="col"
            >
              <div
                id="table-sort-7"
                style="display: none;"
              >
                Click to sort rows by [object Object] header in ascending order
              </div>
              <button
                aria-describedby="table-sort-7"
                class="bx--table-sort"
                data-column-id="wind"
                data-sort-cycle="tri-states-from-ascending"
                data-sort-direction="NONE"
                type="button"
              >
                <span
                  class="bx--table-sort__flex"
                >
                  <div
                    class="bx--table-header-label"
                  >
                    <div
                      style="display: flex; align-items: center; justify-content: space-between;"
                    >
                      <div
                        style="margin-right: 20px;"
                      >
                        Wind
                         
                        <br />
                        <span
                          class="bx--type-helper-text-01"
                        >
                          in km/h
                        </span>
                      </div>
                    </div>
                  </div>
                  <svg
                    aria-hidden="true"
                    class="bx--table-sort__icon"
                    fill="currentColor"
                    focusable="false"
                    height="20"
                    preserveAspectRatio="xMidYMid meet"
                    viewBox="0 0 32 32"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M16 4L6 14 7.41 15.41 15 7.83 15 28 17 28 17 7.83 24.59 15.41 26 14 16 4z"
                    />
                  </svg>
                  <svg
                    aria-hidden="true"
                    class="bx--table-sort__icon-unsorted"
                    fill="currentColor"
                    focusable="false"
                    height="20"
                    preserveAspectRatio="xMidYMid meet"
                    viewBox="0 0 32 32"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M27.6 20.6L24 24.2 24 4 22 4 22 24.2 18.4 20.6 17 22 23 28 29 22zM9 4L3 10 4.4 11.4 8 7.8 8 28 10 28 10 7.8 13.6 11.4 15 10z"
                    />
                  </svg>
                </span>
              </button>
            </th>
            <th
              aria-sort="none"
              scope="col"
            >
              <div
                id="table-sort-8"
                style="display: none;"
              >
                Click to sort rows by [object Object] header in ascending order
              </div>
              <button
                aria-describedby="table-sort-8"
                class="bx--table-sort"
                data-column-id="gust"
                data-sort-cycle="tri-states-from-ascending"
                data-sort-direction="NONE"
                type="button"
              >
                <span
                  class="bx--table-sort__flex"
                >
                  <div
                    class="bx--table-header-label"
                  >
                    <div
                      style="display: flex; align-items: center; justify-content: space-between;"
                    >
                      <div
                        style="margin-right: 20px;"
                      >
                        Gust
                         
                        <br />
                        <span
                          class="bx--type-helper-text-01"
                        >
                          in km/h
                        </span>
                      </div>
                    </div>
                  </div>
                  <svg
                    aria-hidden="true"
                    class="bx--table-sort__icon"
                    fill="currentColor"
                    focusable="false"
                    height="20"
                    preserveAspectRatio="xMidYMid meet"
                    viewBox="0 0 32 32"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M16 4L6 14 7.41 15.41 15 7.83 15 28 17 28 17 7.83 24.59 15.41 26 14 16 4z"
                    />
                  </svg>
                  <svg
                    aria-hidden="true"
                    class="bx--table-sort__icon-unsorted"
                    fill="currentColor"
                    focusable="false"
                    height="20"
                    preserveAspectRatio="xMidYMid meet"
                    viewBox="0 0 32 32"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M27.6 20.6L24 24.2 24 4 22 4 22 24.2 18.4 20.6 17 22 23 28 29 22zM9 4L3 10 4.4 11.4 8 7.8 8 28 10 28 10 7.8 13.6 11.4 15 10z"
                    />
                  </svg>
                </span>
              </button>
            </th>
            <th
              aria-sort="none"
              scope="col"
            >
              <div
                id="table-sort-9"
                style="display: none;"
              >
                Click to sort rows by [object Object] header in ascending order
              </div>
              <button
                aria-describedby="table-sort-9"
                class="bx--table-sort"
                data-column-id="solar"
                data-sort-cycle="tri-states-from-ascending"
                data-sort-direction="NONE"
                type="button"
              >
                <span
                  class="bx--table-sort__flex"
                >
                  <div
                    class="bx--table-header-label"
                  >
                    <div
                      style="display: flex; align-items: center; justify-content: space-between;"
                    >
                      <div
                        style="margin-right: 20px;"
                      >
                        Solar irradiation
                         
                        <br />
                        <span
                          class="bx--type-helper-text-01"
                        >
                          in w/m²
                        </span>
                      </div>
                    </div>
                  </div>
                  <svg
                    aria-hidden="true"
                    class="bx--table-sort__icon"
                    fill="currentColor"
                    focusable="false"
                    height="20"
                    preserveAspectRatio="xMidYMid meet"
                    viewBox="0 0 32 32"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M16 4L6 14 7.41 15.41 15 7.83 15 28 17 28 17 7.83 24.59 15.41 26 14 16 4z"
                    />
                  </svg>
                  <svg
                    aria-hidden="true"
                    class="bx--table-sort__icon-unsorted"
                    fill="currentColor"
                    focusable="false"
                    height="20"
                    preserveAspectRatio="xMidYMid meet"
                    viewBox="0 0 32 32"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M27.6 20.6L24 24.2 24 4 22 4 22 24.2 18.4 20.6 17 22 23 28 29 22zM9 4L3 10 4.4 11.4 8 7.8 8 28 10 28 10 7.8 13.6 11.4 15 10z"
                    />
                  </svg>
                </span>
              </button>
            </th>
            <th
              aria-sort="none"
              scope="col"
            >
              <div
                id="table-sort-10"
                style="display: none;"
              >
                Click to sort rows by [object Object] header in ascending order
              </div>
              <button
                aria-describedby="table-sort-10"
                class="bx--table-sort"
                data-column-id="uvi"
                data-sort-cycle="tri-states-from-ascending"
                data-sort-direction="NONE"
                type="button"
              >
                <span
                  class="bx--table-sort__flex"
                >
                  <div
                    class="bx--table-header-label"
                  >
                    <div
                      style="display: flex; align-items: center; justify-content: space-between;"
                    >
                      <div
                        style="margin-right: 20px;"
                      >
                        UV Index
                         
                        <br />
                        <span
                          class="bx--type-helper-text-01"
                        />
                      </div>
                    </div>
                  </div>
                  <svg
                    aria-hidden="true"
                    class="bx--table-sort__icon"
                    fill="currentColor"
                    focusable="false"
                    height="20"
                    preserveAspectRatio="xMidYMid meet"
                    viewBox="0 0 32 32"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M16 4L6 14 7.41 15.41 15 7.83 15 28 17 28 17 7.83 24.59 15.41 26 14 16 4z"
                    />
                  </svg>
                  <svg
                    aria-hidden="true"
                    class="bx--table-sort__icon-unsorted"
                    fill="currentColor"
                    focusable="false"
                    height="20"
                    preserveAspectRatio="xMidYMid meet"
                    viewBox="0 0 32 32"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M27.6 20.6L24 24.2 24 4 22 4 22 24.2 18.4 20.6 17 22 23 28 29 22zM9 4L3 10 4.4 11.4 8 7.8 8 28 10 28 10 7.8 13.6 11.4 15 10z"
                    />
                  </svg>
                </span>
              </button>
            </th>
          </tr>
        </thead>
        <tbody
          aria-live="polite"
        >
          <tr
            data-row-id="2"
          >
            <td>
              2021/03/05 03:57
            </td>
            <td>
              0.4
            </td>
            <td>
              0.4
            </td>
            <td>
              0.2
            </td>
            <td>
              986.1
            </td>
            <td>
              99
            </td>
            <td>
              0
            </td>
            <td>
              0
            </td>
            <td>
              0
            </td>
            <td>
              0
            </td>
          </tr>
          <tr
            data-row-id="3"
          >
            <td>
              2021/03/05 05:57
            </td>
            <td>
              -0.2
            </td>
            <td>
              -0.2
            </td>
            <td>
              -0.7
            </td>
            <td>
              987.6
            </td>
            <td>
              96
            </td>
            <td>
              1.4
            </td>
            <td>
              3.6
            </td>
            <td>
              0
            </td>
            <td>
              0
            </td>
          </tr>
          <tr
            data-row-id="4"
          >
            <td>
              2021/03/05 07:57
            </td>
            <td>
              -0.7
            </td>
            <td>
              -0.7
            </td>
            <td>
              -1.8
            </td>
            <td>
              989.7
            </td>
            <td>
              93
            </td>
            <td>
              1.1
            </td>
            <td>
              1.8
            </td>
            <td>
              38.1
            </td>
            <td>
              0
            </td>
          </tr>
          <tr
            data-row-id="5"
          >
            <td>
              2021/03/05 09:57
            </td>
            <td>
              1.2
            </td>
            <td>
              1.2
            </td>
            <td>
              -2.4
            </td>
            <td>
              991
            </td>
            <td>
              77
            </td>
            <td>
              0
            </td>
            <td>
              0
            </td>
            <td>
              154.3
            </td>
            <td>
              1
            </td>
          </tr>
          <tr
            data-row-id="6"
          >
            <td>
              2021/03/05 11:57
            </td>
            <td>
              2.4
            </td>
            <td>
              2.4
            </td>
            <td>
              -5
            </td>
            <td>
              992.2
            </td>
            <td>
              58
            </td>
            <td>
              0.4
            </td>
            <td>
              1.8
            </td>
            <td>
              106.6
            </td>
            <td>
              1
            </td>
          </tr>
        </tbody>
      </table>
    </div>
    <div
      class="bx--pagination"
    >
      <div
        class="bx--pagination__left"
      >
        <label
          class="bx--pagination__text"
          for="bx-pagination-select-1"
          id="bx-pagination-select-1-count-label"
        >
          Items per page:
        </label>
        <div
          class="bx--form-item"
        >
          <div
            class="bx--select bx--select--inline bx--select__item-count"
          >
            <div
              class="bx--select-input--inline__wrapper"
            >
              <div
                class="bx--select-input__wrapper"
              >
                <select
                  class="bx--select-input"
                  id="bx-pagination-select-1"
                >
                  <option
                    class="bx--select-option"
                    value="10"
                  >
                    10
                  </option>
                  <option
                    class="bx--select-option"
                    value="25"
                  >
                    25
                  </option>
                  <option
                    class="bx--select-option"
                    value="50"
                  >
                    50
                  </option>
                  <option
                    class="bx--select-option"
                    value="100"
                  >
                    100
                  </option>
                  <option
                    class="bx--select-option"
                    value="200"
                  >
                    200
                  </option>
                  <option
                    class="bx--select-option"
                    value="500"
                  >
                    500
                  </option>
                </select>
                <svg
                  aria-hidden="true"
                  class="bx--select__arrow"
                  fill="currentColor"
                  focusable="false"
                  height="16"
                  preserveAspectRatio="xMidYMid meet"
                  viewBox="0 0 16 16"
                  width="16"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M8 11L3 6 3.7 5.3 8 9.6 12.3 5.3 13 6z"
                  />
                </svg>
              </div>
            </div>
          </div>
        </div>
        <span
          class="bx--pagination__text bx--pagination__items-count"
        >
          1–5 of 5 items
        </span>
      </div>
      <div
        class="bx--pagination__right"
      >
        <div
          class="bx--form-item"
        >
          <div
            class="bx--select bx--select--inline bx--select__page-number"
          >
            <label
              class="bx--label bx--visually-hidden"
              for="bx-pagination-select-1-right"
            >
              Page number, of 1 pages
            </label>
            <div
              class="bx--select-input--inline__wrapper"
            >
              <div
                class="bx--select-input__wrapper"
              >
                <select
                  class="bx--select-input"
                  id="bx-pagination-select-1-right"
                >
                  <option
                    class="bx--select-option"
                    value="1"
                  >
                    1
                  </option>
                </select>
                <svg
                  aria-hidden="true"
                  class="bx--select__arrow"
                  fill="currentColor"
                  focusable="false"
                  height="16"
                  preserveAspectRatio="xMidYMid meet"
                  viewBox="0 0 16 16"
                  width="16"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M8 11L3 6 3.7 5.3 8 9.6 12.3 5.3 13 6z"
                  />
                </svg>
              </div>
            </div>
          </div>
        </div>
        <span
          class="bx--pagination__text"
        >
          of 1 page
        </span>
        <div
          class="bx--pagination__control-buttons"
        >
          <button
            class="bx--pagination__button bx--pagination__button--backward bx--pagination__button--no-index bx--btn bx--btn--ghost bx--btn--disabled bx--tooltip--hidden bx--btn--icon-only bx--tooltip__trigger bx--tooltip--a11y bx--btn--icon-only--top bx--tooltip--align-center"
            disabled=""
            tabindex="0"
            type="button"
          >
            <div
              class="bx--assistive-text"
            >
              Previous page
            </div>
            <svg
              aria-hidden="true"
              aria-label="Previous page"
              class="bx--btn__icon"
              fill="currentColor"
              focusable="false"
              height="16"
              preserveAspectRatio="xMidYMid meet"
              role="img"
              viewBox="0 0 32 32"
              width="16"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M20 24L10 16 20 8z"
              />
            </svg>
          </button>
          <button
            class="bx--pagination__button bx--pagination__button--forward bx--pagination__button--no-index bx--btn bx--btn--ghost bx--btn--disabled bx--tooltip--hidden bx--btn--icon-only bx--tooltip__trigger bx--tooltip--a11y bx--btn--icon-only--top bx--tooltip--align-end"
            disabled=""
            tabindex="0"
            type="button"
          >
            <div
              class="bx--assistive-text"
            >
              Next page
            </div>
            <svg
              aria-hidden="true"
              aria-label="Next page"
              class="bx--btn__icon"
              fill="currentColor"
              focusable="false"
              height="16"
              preserveAspectRatio="xMidYMid meet"
              role="img"
              viewBox="0 0 32 32"
              width="16"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M12 8L22 16 12 24z"
              />
            </svg>
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
